ckanext/unhcr/templates/package/resource_read.html

```diff
@@ -68,7 +68,37 @@
 
 {% endblock %}
 
-{% block secondary_content %}
+{% block primary_content %}
+
+{{ super() }}
+
+{% if res.kobo_type %}
+<section class="module">
+  
+  <div class="module-content">
+    <h2>{{ _('KoBo information') }}</h2>
+    <table class="table table-striped table-bordered table-condensed" data-module="table-toggle-more">
+      <tbody>
+        <tr>
+          <th scope="row">{{ _('KoBo asset ID') }}</th>
+          <td>{{ c.package.kobo_asset_id }}</td>
```

Comment:
Unless strictly necessary we should no longer use variables in the `c` object in templates. In 2.9 all core properties added to `c` are now passed directly to templates:

```suggestion
          <td>{{ package.kobo_asset_id }}</td>
```